ui/src/frontend/panel_container.ts

```diff
@@ -237,32 +239,60 @@ export class PanelContainer implements m.ClassComponent<Attrs> {
     return (attrs as {collapsed?: boolean}).collapsed !== undefined;
   }
 
-  renderPanel(node: AnyAttrsVnode, key: string, extraClass = ''): m.Vnode {
+  renderPanel(node: AnyAttrsVnode, key: string, style?: object): m.Vnode {
```

Comment:
nit: I would prefer that the `style` parameter be typed as `CSSStyleDeclaration` for type safety in maintenance of the style attributes. Note that this would require adapting the `'z-index'` style attribute name as `zIndex` and converting its value to a string. It works just as well when I try this change locally 😀